.infrastructure/deployment.yml

```
@@ -29,7 +29,15 @@ spec:
             cpu: "150m"
         env:
         - name: PYTHONUNBUFFERED
-          value: "1"
+          valueFrom:
```

This `valueFrom:` is used to source environment variables from Kubernetes resources, which is correct. Optionally, you can add `optional: false` under the reference to explicitly cause scheduling failure when the referenced resource is missing, making missing-config issues fail fast.